CMakeLists.txt

```diff
@@ -19,8 +19,12 @@ list(APPEND CMAKE_PREFIX_PATH ${ARMONIK_API_DIR})
 if (WIN32)
     list(APPEND CMAKE_PREFIX_PATH "${CMAKE_SOURCE_DIR}/tools/win64/lib/cmake/grpc")
     list(APPEND CMAKE_PREFIX_PATH "${CMAKE_SOURCE_DIR}/tools/win64/cmake")
+    set(INSTALL_SDK_DIR ${CMAKE_SOURCE_DIR}/install)
     set(Protobuf_USE_STATIC_LIBS ON)
+else()
+    set(INSTALL_SDK_DIR "/app/install")
```

Comment:
Should not be hard coded. ${CMAKE_SOURCE_DIR}/install would be fine here I think

Reply:
Actually, the install directory is outside the source directory in the docker.